examples/1-basic/density.ipynb

```diff
@@ -66,7 +66,11 @@
     "shells = glass.linear_windows(zb)\n",
     "\n",
     "# compute the angular matter power spectra of the shells with CAMB\n",
-    "cls = glass.ext.camb.matter_cls(pars, lmax, shells)"
+    "cls = glass.ext.camb.matter_cls(\n",
+    "    pars,\n",
+    "    lmax,\n",
+    "    [(s.za, s.wa, s.zeff) for s in shells],\n",
```

Comment:
This feels like a silly hack. Is it possible to change `glass.ext.camb.matter_cls` instead?